inst/qml/jaspPlotBuilder.qml

```diff
@@ -0,0 +1,2901 @@
+                                label: qsTr("Column")
+                                source: [ { name: "columnsvariableSplitPlotBuilder", use: "levels" } ]
+                            }
+
+                            DropDown {
+                                name: "RowPValue"
+                                label: qsTr("Row")
+                                source: [ { name: "rowsvariableSplitPlotBuilder", use: "levels" } ]
+                            }
+
+                        }
+
+                    }
+                }
+            }
+        }
+    }
+
+    Section {
+        title: qsTr("Plot layout")
+        columns: 1
+
+        Group {
+            title: qsTr("Arrange plots by column")
+
+            // Relative width of the columns
+            TextField {
+                name: "columnWidthInput"
+                label: qsTr("Relative column widths")
+                placeholderText: qsTr("1,1")
+            }
+
+            // Row Specifications ComponentsList
+            ComponentsList {
+                id: rowSpecifications
+                name: "rowSpecifications"
+                title: qsTr("Specify layout")
+                addItemManually: true
+                newItemName: qsTr("Column 1")
+                Layout.preferredWidth: form.width - 2 * jaspTheme.generalAnchorMargin
+
+                rowComponent: Row {
+                    Group {
+                        title: qsTr("Column ") + (rowIndex + 1)
+
+                        TextField {
+                            name: "plotIDs"
+                            label: qsTr("Plot IDs")
+                            placeholderText: qsTr("Plot 1, Plot 2, ...")
+                        }
+
+                        TextField {
+                            name: "rowHeightsColumn"
+                            label: qsTr("Plot heights")
+                            info: qsTr("Specify the relative heights of the plots in the specified column.")
+                            placeholderText: qsTr("1,1")
+                        }
+                    }
+
+                    Group {
+                        title: " "
+
+                        TextField {
+                            name: "labelsColumn"
+                            label: qsTr("Labels")
+                            placeholderText: qsTr("A, B, C, ...")
+                            fieldWidth: 150
+                        }
+
+                        CheckBox {
+                            name: "getCommonLegendColumn"
+                            label: qsTr("Collect legend")
+                        }
+                    }
+                }
+            }
+        }
+
+        Group {
+            title: qsTr("Arrange plots by row")
+
+            // Relative height within row layout
+            TextField {
+                name: "relHeightWithinRowLayout"
+                label: qsTr("Relative row heights")
+                placeholderText: qsTr("1,1")
```

Comment:
```suggestion
                placeholderText: "1,1"
```

Another minor translation change: remove unnecessary qsTr() cause we dont want to translate digits:-)